### ISSUE TITLE
Explain the Cost/Benefits of Time-Estimating Stories (Before Starting)

### ISSUE THREAD

**Issue by nelsonic** [tcm-labs/workflow#8]
+ [ ] Why should we estimate our stories?
  + [ ] How long does it take to estimate a story? > _Less than **1 minute** per story_.
+ [ ] What are the Advantages of Time Estimation?
  + [ ] _Confirm_ your _understanding_ of the story. 
  + [ ] _Confirm_ that the story is not "_too big_" (_otherwise split it out into smaller sub-stories_)
  + [ ] _Commit_ to getting the work done in a _finite_ amount of time.
  + [ ] _Set_ a time limit on getting the work done (_so you ask for help from peers faster if your are stuck!_)
  + [ ] Stay focussed on delivering _exactly_ what is in the story (_acceptance criteria/checklist_) without getting distracted or side-tracked.
+ [ ] How do we apply a Time Estimate Label?
+ [ ] Who should _estimate_ the story?
